spacewalk/package/spacewalk.changes

```diff
@@ -1,3 +1,5 @@
+- require postgresql 9.6 as we only support this version for
+  migration to version 10. Take into account new versioning scheme
```

Comment:
Please add `(FATE#325659)` to the migration to 10. I like to have numbers if we have them